docs/accounts/switching.md

```diff
@@ -0,0 +1,17 @@
+# Switching Between Multiple Accounts
```

Comment:
If you want to use title casing for headers, please create an issue so we don't forget to change existing files to also use it.
(Personally, I'm not a fan.)

Reply:
fixed